### DISCUSSION TITLE
[Feature] Static web share page

### DISCUSSION THREAD

**Discussion by chrislxn** [immich-app/immich#13138] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [x] Yes

### The feature

I'm a photographer and I sometimes share my albums to the public. But every time when A new visitor open the share link, lots of resources are loaded from the server, which requires lots of processes on the server. I tried to use the Cloudflare Cache, but sometimes it returns a 403 error.

If there could be a feature that a static page could be generated for public shared albums so that It will save a lot of hardware resources, just like a gallery. The cache could be purged only when the albums changed.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Making the page static wouldn't really make a difference in the serverside resource usage. Frankly, if the load of viewing a shared link is too much, then your system is not powerful enough for Immich.
